crates/bevy_input_focus/src/lib.rs

```diff
@@ -135,6 +138,13 @@ fn dispatch_keyboard_input(
     }
 }
 
+/// System which synchronizes the a11y focus state with the input focus state.
+fn sync_a11y_focus(focus: Res<InputFocus>, mut a11y_focus: ResMut<Focus>) {
```

Comment:
This seems silly: shouldn't we just change the a11y focus to use the new resource?

Reply:
Uh, I did it this way because my understanding was (based on our previous discussion) that we couldn't have `bevy_a11y` depend on `bevy_input_focus`.

Reply:
Hmm. Maybe I'm misremembering or maybe there was a micommunication, but bevy_a11y depending on bevy_input_focus definitely seems more correct than the reverse (like it is now).